src/jit/morph.cpp

```diff
@@ -1367,7 +1367,7 @@ void fgArgInfo::RemorphStkArg(
     nextSlotNum += numSlots;
 }
 
-void fgArgInfo::SplitArg(unsigned argNum, unsigned numRegs, unsigned numSlots)
+void fgArgInfo::SplitArg(unsigned argNum, unsigned numRegs, unsigned numSlots, bool isReMorph)
```

Comment:
It is quite unfortunate that this method has the side-effect of updating the stateful field `nextSlotNum`. Otherwise I would suggest avoiding the call to this method when remorphing.